src/platform/packages/private/kbn-index-editor/src/components/query_bar.tsx

```diff
@@ -45,10 +52,20 @@ export const CustomPanel = () => {
 
   return (
     <EuiFlexGroup alignItems={'center'} gutterSize={'s'}>
+      <EuiFlexItem grow>
+        {dataView ? (
+          <EuiSearchBar
+            defaultQuery={''}
+            onChange={({ queryText, query, error }) => {
```

Comment:
```suggestion
            onChange={({ queryText }) => {
```

Or should we display this error? What kind of errors can this component return?

Reply:
Updated in https://github.com/elastic/kibana/pull/230140/commits/fce4d4476f0d6ac5cfa447bdb0f0a83880f36be3